curator/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel8.2.els.python.36
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-base-openshift-4.6
```

Comment:
can't we use `registry.redhat.io/ubi8/python-38:latest` ? then we wont have to install python

Reply:
OCP4.6 is based on RHEL 8.2 EUS. By choosing ubi8/python* we'd deviate from the requirement.